### PR TITLE
Enhance input form with description and checkbox

### PR DESCRIPTION
**Pull request by nfode**

### CODE REVIEW

File: app/src/app/components/input/input.component.html [hunk 1]
```diff
@@ -16,10 +16,18 @@
         <tr *ngFor="let item of inputs; let n=index">
             <td>{{item.key}}</td>
             <td>{{item.type}}</td>
-            <td><input [value]="item.value"  (input)="change(item, $event.target.value)" name="item.key"
-                       [ngClass]="{'invalid': !item.valid, 'valid':item.valid}"/>
+            <td *ngIf="!hasTypeBoolean(item)"><input [value]="item.value"
```
Comment (c-mueller) [inline app/src/app/components/input/input.component.html]: Please consider Reformatting this.

File: app/src/app/components/input/input.component.scss [hunk 2]
```diff
@@ -42,4 +42,14 @@ form {
             margin: 2px;
         }
     }
+    input[type=checkbox] {
```
Comment (c-mueller) [inline app/src/app/components/input/input.component.scss]: Why does github mark this red?

Are you sure this is syntactically correct?

Comment (nfode) [reply app/src/app/components/input/input.component.scss]: It is correct. this is a so called attribute selector. [Check this link.](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors)

Comment (c-mueller) [reply app/src/app/components/input/input.component.scss]: Okay. Was just wondering why this is highlighted red

**Pull request merged**